modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ManageProductUsers/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {Button} from '../../../../common/components';
+
+const ManageProductUser = ({activationStatusAC, activationStatusDXPC}) => {
```

Comment:
Might be better pass the subscriptionGroups as prop and make the validation inside this component

```suggestion
const ManageProductUser = ({subscriptionGroups}) => {
```

Reply:
Ok, done.